lighthouse-plots/analyze.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const utils = require('./utils');
+
+const OUT_PATH = path.resolve(__dirname, 'out');
+const GENERATED_RESULTS_PATH = path.resolve(OUT_PATH, 'generatedResults.js');
+const LIGHTHOUSE_RESULTS_FILENAME = 'lighthouse.json';
+
+const METRICS = [
+  'firstMeaningfulPaint',
+  'speedIndexMetric',
+  'estimatedInputLatency',
+  'timeToInteractive',
+  'visuallyReady',
+  'traceEnd'
+];
+
+// TODO: annotate
+function main() {
+  const allResults = [];
+  fs.readdirSync(OUT_PATH).map(siteDir => {
```

Comment:
Yeah - it's like how Chromium uses out/ to store all generated files and sometimes uses out/ as inputs to generate additional files.